.github/workflows/packaging.yml

```diff
@@ -37,12 +37,12 @@ jobs:
           cd build
           make package_source
       - name: Rename tarball
-        run: mv build/omniperf-*.tar.gz build/omniperf-${{github.ref_name}}.tar.gz
+        run: mv build/rocprof-compute-*.tar.gz build/rocprof-compute-${{github.ref_name}}.tar.gz
       - name: Archive tarball
         uses: actions/upload-artifact@v4
         with:
-          name: omniperf-${{github.ref_name}}.tar.gz
-          path: build/omniperf-${{github.ref_name}}.tar.gz
+          name: rocprof-compute-${{github.ref_name}}.tar.gz
+          path: build/rocprof-compute-${{github.ref_name}}.tar.gz
       - name: Set version
         run: echo "VERSION=$(cat VERSION)" >> $GITHUB_ENV
       - name: Determine release name
```

Comment:
The `RELEASE_NAME` should be changed too. 
It's just below here. The review would not let me update it with a suggestion.